AutomatedTesting/Gem/PythonTests/largeworlds/gradient_signal/TestSuite_Periodic_Optimized.py

```diff
@@ -10,6 +10,7 @@
 from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorSharedTest, EditorParallelTest, EditorTestSuite
 
 
+@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
```

Comment:
I believe the CTest default output will suppress showing the logs containing xfail results.
Where are these being monitored, locally? Or does MARS separate xfail reports?

Reply:
Monitored via MARS